exonum/src/blockchain/tests/mod.rs

```diff
@@ -165,7 +167,7 @@ impl Transaction {
     ) -> Verified<AnyTx> {
         let tx = AnyTx {
             arguments: self.into_bytes(),
-            call_info: CallInfo::new(instance_id, 0),
+            call_info: CallInfo::new(instance_id, 0, ""),
```

Comment:
It could make sense to retain the old constructor and add a new constructor for an interface call (say, `CallInfo::with_interface`).